payjoin/src/receive/v1/mod.rs

```diff
@@ -847,6 +845,42 @@ pub(crate) mod test {
         }
     }
 
+    #[test]
+    fn empty_candidates_inputs() -> Result<(), BoxError> {
```

Comment:
You can take advantage of the BoxError here by using `?` to propagate errors instead of calling `.expect`. Doing so will make the test more brief and readable.

A followup commit might make this change to use `BoxError` in the other tests in this module.